tests/Response.test.ts

```diff
@@ -680,28 +689,50 @@ describe('Response', () => {
       });
 
       describe('jsonp', () => {
-         const test = (evt: RequestEvent, msg: string | false, extender?: Extender, overrideQueryParamName?: string | false): void => {
-            let o = { foo: 'bar' },
-                queryParamName = overrideQueryParamName || 'callback',
-                body = `/**/ typeof fooFunction === 'function' && fooFunction(${JSON.stringify(o)});`;
+         const test = (
```

Comment:
I'm not too thrilled with the increase in number of code paths through this test function. However, don't have any good ideas to improve the situation at the moment. Thoughts?

Reply:
Yeah, this went from bad to atrocious. Couldn't the `expectations`, at least, have been prevented by using extenders? That was the point of the test extender.